config/locales/en.yml

```diff
@@ -819,12 +820,14 @@ en:
       media:
         title: Media
       metadata: Metadata
+      no_history: This post hasn't been edited
       no_status_selected: No posts were changed as none were selected
       open: Open post
       original_status: Original post
       reblogs: Reblogs
       status_changed: Post changed
-      title: Account posts
+      status_title: Post by @%{name}
+      title: Account posts - @%{name}
```

Comment:
This change will require all translations for `admin.statuses.title` to be updated, so may be contentious, however, I think changing to have the account name in the localisation will allow for better localisations.